File: GCB-Investimentos/Node.js_API/Models/tb_usuarios.js
class Usuario{
    constructor(Id,Nome,Email,Senha,DataNascimento,Cpf,IdTipoUsuario){
       this.Id = Id;
       this.Nome = Nome;
       this.Email = Email;
       this.Cpf = Cpf;
       this.Senha = Senha;
       this.IdTipoUsuario = IdTipoUsuario;

    }
}

module.exports = Usuario;

File: GCB-Investimentos/GCB_Care_WebApi/Models/TbUsuarios.cs
using System;
using System.Collections.Generic;

namespace GCB_Care_WebApi.Models
{
    public partial class TbUsuarios
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Cpf { get; set; }
        public string Senha { get; set; }
        public int? IdTipoUsuario { get; set; }

        public TbTipoUsuario IdTipoUsuarioNavigation { get; set; }
    }
}
